conformance_tests/sysman/test_sysman_memory/src/test_sysman_memory.cpp

```diff
@@ -443,4 +462,91 @@ TEST_F(
     memoryThread.join();
   }
 }
+
+void getMemoryStateTemp(ze_device_handle_t device) {
+  std::cout<<"\nInside function getMemoryStateTemp";
+  uint32_t count = 0;
+  std::vector<zes_mem_handle_t> mem_handles =
+      lzt::get_mem_handles(device, count);
+  if (count == 0) {
+    FAIL() << "No handles found: "
+           << _ze_result_t(ZE_RESULT_ERROR_UNSUPPORTED_FEATURE);
+  }
+  for (auto mem_handle : mem_handles) {
+    ASSERT_NE(nullptr, mem_handle);
+    lzt::get_mem_state(mem_handle);
+  }
+  std::cout<<"\nExiting function getMemoryStateTemp";
+}
+
+void getFirmwareProperties(zes_firmware_handle_t firmware_handle, zes_device_properties_t deviceProperties) {
+  std::cout<<"\nCalling function getFirmwareProperties";
+  ASSERT_NE(nullptr, firmware_handle);
+  auto properties = lzt::get_firmware_properties(firmware_handle);
+  //Are the below lines till line.495 required? 
```

Comment:
Comments which are not required can be removed, before rising for review.

Reply:
But I was not sure if I should include/ exclude the lines which verifies if retrieved firmware properties are valid